### DISCUSSION TITLE
How to contribute in Google Summer of Code

### DISCUSSION THREAD

**Discussion by SusheelThapa** [Schweinepriester/github-profile-achievements#650] [Q&A] [answered]
How to contribute in Google Summer of Code?

**Comment by ParanoidUser** [comment, marked as answer]
Simply put, there are two ways to participate in GSoC:
1. If you're the maintainer of a repo that might attract other people, you can register as an organization and mentor other developer.
   **Note:** Registration for orgs already closed for this year. They will announce the full list of eligible orgs on February 22nd.
2. If you're willing to contribute to other projects, you can send contributor application.
   **Note:** Registration starts on March 20 to April 4.

Keep in mind, when you submit application, you're applying for a "*potential GSoC contributor*". There will be 2 rounds after that:
- Round 1: contact the mentor organizations and write a project proposal based on ideas the organization has suggested. If accepted, GSoC contributors spend a few weeks becoming familiar with the community norms and codebase while determining expected milestones with their mentor for the summer. 
- Round 2: spend 12+ weeks coding on their projects.

Learn more - https://summerofcode.withgoogle.com/get-started
